Card.py

```diff
@@ -0,0 +1,12 @@
+import random
+
+
+class Card:
+    def __init__(self):
+        colors = ["Red", "Blue", "Green", "Yellow"]
+        self.num = random.randrange(1, 10)
+        self.color = random.choice(colors)
+
+    def __eq__(self, other):
+        return self.__dict__ == other.__dict__
```

Comment:
Do you know what __dict__ does?

Reply:
Well basically it creates a dictionary that stores object attributes.
Am I missing something?